geniza/footnotes/admin.py

```diff
@@ -110,12 +110,21 @@ class FootnoteAdmin(admin.ModelAdmin):
         }),
         (None, {
             'fields': (
-                'source', 'location', 'doc_relation',
-                'notes'
+                'source', 'location', 'doc_relation', 'content_blocks', 'notes'
             )
         })
     ]
 
+    def content_blocks(self, obj):
+        output = ""
+        for block in obj.content["blocks"]:
+            if block["label"]:
+                output += f"{block['label']}\n"
+            for line in block["lines"]:
+                output += f"{line}\n"
+        return output
+    content_blocks.short_description = "Content"
```

Comment:
tried using some light html here but it didn't render; maybe `mark_safe()` is needed or something?

Reply:
I think we should probably do it in the html so we can right align and set rtl text direction. (Although it seems to already be displaying rtl? does the browser pick that up automatically?)  

I suggest setting rtl/right align for everything right now rather than trying to figure out what script it is.

Reply:
change template seems like it would give more opportunity to customize; I'll try that